README.md

```diff
@@ -34,6 +34,17 @@ cd example/
 php ../bin/stakx build
 ```
 
+### Locally Serving a Site
+
+Stakx includes a built in PHP server, which you can run with the following commands. Access your site at `http://0.0.0.0:8000/`.
```

Comment:
```suggestion
Stakx includes a built-in web server, which you can run with the following commands. Access your site at `http://0.0.0.0:8000/`.
```

"PHP server" makes it sound like it'll execute PHP scripts, which is not the case.

Reply:
Thanks — change made.